api/src/App/Dependencies.php

```diff
@@ -43,4 +47,16 @@ function setupAPIDependencies(App $app, array $settings)
         );
     };
 
+    $container['EventService'] = function ($c): EventService {
+        return new EventService(
+            new EventRepository($c['db'])
+        );
+    };
+
+    $container['MemberService'] = function ($c): MemberService {
+        return new MemberService(
+            new MemberRepository($c['db'])
+        );
+    };
+
 }
```

Comment:
I think these should go in the pervious patches that define the `Event` and `Member` stack (Repository and Service)  I feel like it makes more send there than to have this separate patch to hook them in.

Reply:
Sure, no problem. I can make the adjustment there.